build/edge/README.md

```diff
@@ -27,13 +27,13 @@ container and MQTT Broker, so make sure that docker engine listening on
   ```shell
   ./build/edge/run_daemon.sh set \
   		    cloudhub=0.0.0.0:10000 \
```

Comment:
Please remove some unnecessary whitespaces in this line?

Reply:
I guess we need it like the following line ` Optional: x86_64 \| aarch` :)

Reply:
Done